src/Common/src/Interop/Windows/Kernel32/Interop.QueryPerformanceCounter.cs

```diff
@@ -2,14 +2,21 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Runtime.InteropServices;
 
 internal partial class Interop
 {
     internal partial class Kernel32
     {
-        [DllImport(Libraries.Kernel32)]
-        internal static extern bool QueryPerformanceCounter(out long value);
+        // The actual native signature is:
+        //      BOOL WINAPI QueryPerformanceCounter(
+        //          _Out_ LARGE_INTEGER* lpPerformanceCount
+        //      );
+        //
+        // We take a long* (rather than a out long) to avoid the pinning overhead.
+        // We don't set last error since we don't need the extended error info.
+
+        [DllImport(Libraries.Kernel32, BestFitMapping = false, ExactSpelling = true)]
```

Comment:
`BestFitMapping = false` is unnecessary

Reply:
Isn't true the default? Does it have no impact if you only have ASCII characters?

Reply:
Right, it does nothing unless your PInvoke has `Unicode<->ANSI` marshaling.

Reply:
Or if you are consuming from ProjectN code, if I recall correctly it is overloaded to mean "ignore allow list at ILC time"

Reply:
That is `ExactSpelling`, not `BestFitMapping`.